kit/ForKit.cpp

```diff
@@ -345,6 +362,24 @@ static void cleanupChildren()
         }
     }
 
+    if (killedCount)
+    {
+        if (WSHandler)
+        {
+            std::stringstream stream;
+            stream << "killedcount " << killedCount << '\n';
+            int ret = WSHandler->sendMessage(stream.str());
```

Comment:
Why is this new / duplicate code - surely we could share / re-factor the other use-case ? I expect we could send a single message with three counts in it and reduce code duplication.

Reply:
Now using a lambda.

Reply:
Now use a simple loop to handle this.